src/components/Header/Header.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import AuthContext from '../../context/AuthContext';
+import './Header.css';
+
+export default class Header extends Component {
+  static contextType = AuthContext;
+
+  handleLogoutClick = () => {
+    // TODO: log user out
+    this.context.setLoggedInState(false);
+    this.context.setShelterAdminState(false);
+  }
+
+  renderLoginLink() {
```

Comment:
Genuine question: Should this be combined with renderLogoutLink() and have the code logic here, instead of the render at line 36? I don't know enough about React to determine which one would be better in the long run. What are your thoughts?

```
determineLoginLink() {
    this.context.isLoggedIn
        ? return(<Link>LogOut</Link>)
        : return(<Link>LogIn</Link>)
}
```

Reply:
I think we can do it either way and I honestly can't tell which is better practice. I separated renderLoginLink() and renderLogoutLink() because we need a handler only for the logout, so I felt it's more clear to separate them. (If you click logout, you're logged out immediately so we need to call the API and update the context, but login only takes you to the login page and the rest will be handled there.)

Reply:
Alright in that case I think we can leave it as-is and refactor later if we want!